recipes/R/Retail - GradientBoosting/Dockerfile

```diff
@@ -1,8 +1,26 @@
-FROM adobe/acp-dsw-ml-runtime-r:0.13.1
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2019 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+FROM adobe/acp-dsw-ml-runtime-r:0.15.0
 
 RUN install2.r gbm
```

Comment:
In R is very difficult to specify exact version to install. Especially for packages such as gbm and tidyverse, because they bring up some other packages as dependencies. 
It will need command like this : RUN R -e "install.packages('https://cran.rstudio.com/src/contrib/tidyverse_1.2.1.tar.gz', dependencies=TRUE)", but it requires to install packages 'broom', 'dplyr', 'dbplyr', 'forcats', 'haven', 'modelr', 'readxl', 'reprex', 'rvest', 'tidyr manyally before that command. Otherwise, we get 
ERROR: dependencies 'broom', 'dplyr', 'dbplyr', 'forcats', 'haven', 'modelr', 'readxl', 'reprex', 'rvest', 'tidyr' are not available for package 'tidyverse'.
I'm leaving things as they are for now.